Animations/vendor/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+# Vendored libraries
+
+# This file acts as a proxy between all the vendored libraries
+# and the CMake build system.
+
+###############
+# CMake setup #
+###############
+include(ExternalProject)
+
+############
+# cppUtils #
+############
+add_library(CppUtils INTERFACE)
+target_include_directories(CppUtils INTERFACE "${CMAKE_CURRENT_LIST_DIR}/cppUtils/single_include")
+
+########
+# GLFW #
+########
+set(GLFW_BUILD_DOCS OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
+add_subdirectory(GLFW)
+
+############
+# Freetype #
+############
+add_subdirectory(freetype)
+
+##########
+# OpenAL #
+##########
+set(ALSOFT_UTILS OFF)
+set(ALSOFT_NO_CONFIG_UTIL ON)
+set(ALSOFT_EXAMPLES OFF)
+set(ALSOFT_INSTALL OFF)
+set(ALSOFT_INSTALL_CONFIG OFF)
+set(ALSOFT_INSTALL_HRTF_DATA OFF)
+set(ALSOFT_INSTALL_AMBDEC_PRESETS OFF)
+set(ALSOFT_INSTALL_EXAMPLES OFF)
+set(ALSOFT_INSTALL_UTILS OFF)
+add_subdirectory(openal)
+
+########
+# Luau #
+########
+set(CMAKE_CXX_STANDARD 17) # Luau doesn't build with C++20
```

Comment:
I tried upgrading `CppUtils`, but now memory allocation is causing issues. For reference, I upgraded to commit https://github.com/ambrosiogabe/CppUtils/commit/eb6ad35d406178ea7ee40e629183c16955eb18e3 and added `g_logger_init();` to the main. There seems to be some issue related to leak tracking.

Without upgrading CppUtils this can't be reverted as the build then fails on Windows. Might be worth fixing CppUtils (if its a bug there in the first place) before merging this PR then.

Reply:
Hey thanks for the update! I'll take a look at CppUtils and fix up whatever the issue is then upgrade this appropriately when that's fixed 👍